assets/icons/file_icons/cpp.svg

```diff
@@ -0,0 +1,9 @@

```

Comment:
We'll need to change this to be a true SVG.

Right now it looks like the SVG has a PNG embedded inside of it, which isn't what we want.